python/setup.py

```diff
@@ -73,6 +74,9 @@ def find_pybind_headers_dir():
           'correctly'.format(NGRAPH_CPP_DIST_DIR))
     sys.exit(1)
 
+NGRAPH_CPP_LIBRARY_NAME = 'ngraph'
+if len([fn for fn in os.listdir(NGRAPH_CPP_LIBRARY_DIR) if re.search('ngraphd', fn)]):
```

Comment:
Please add a comment explaining why we need to do this and where the name `ngraphd` comes from.

Reply:
It is happened because for OpenVINO package on some platforms we provided Debug and Release binary files and in this case debug files have postfix `d`

Reply:
Yes, please add this as a comment to the code for future reference.

Reply:
Done